.github/workflows/e2e_matrix.json

```diff
@@ -0,0 +1,14 @@
+[
```

Comment:
what do we gain from processing this json file each time instead of having the value hardcoded in the action?

Reply:
Nothing really, a bit more clarity in the file, easier to read